### PR TITLE
Add calour recipe

### PR DESCRIPTION
**Pull request by RNAer**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by RNAer** [comment]
@bioconda/build-system , Anyone know what's going on? there is no circleci code in the recipe - why did it complain about that?

```
#!/bin/bash -eo pipefail
.circleci/setup.sh
From ssh://github.com/bioconda/bioconda-recipes
 * [new branch]          master     -> __upstream__zkhcfpyirb/master
The CI configuration is out of date.
Please merge in bioconda:master.
Exited with code 1
```

**Comment by epruesse** [comment]
You just need to do what it says: `Please merge in bioconda:master.` - you forked bioconda-recipes/master too long ago. If you merge (or rebase + force push) your branch you should be ok.

**Comment by RNAer** [comment]
Thanks, @epruesse ! now passed. @bioconda/core , anyone can help with reviewing and merging?

**Comment by RNAer** [comment]
I guess I should pin @bioconda/core for review

File: recipes/calour/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
```
[the remark this answers is not in the document]
Comment (RNAer) [reply recipes/calour/meta.yaml]: thanks for your comments, @epruesse! why `python <3`?  this package does not work on py2

[the remark this answers is not in the document]
Comment (RNAer) [reply recipes/calour/meta.yaml]: a friendly reminder, @epruesse . really appreciate if you can elaborate a bit more. thanks.

[the remark this answers is not in the document]
Comment (epruesse) [reply recipes/calour/meta.yaml]: @RNAer Sorry, the other way round. Should be `python >=3.5`

**Pull request merged**